corehq/apps/userreports/operators.py

```diff
@@ -6,6 +6,7 @@
 NOT_EQUAL = lambda input, reference: input != reference
 IN = lambda input, reference_list: input in reference_list
 IN_MULTISELECT = lambda input, reference: reference in (input or '').split(' ')
+ANY_IN_MULTISELECT = lambda input, reference: any([subval in (input or '').split(' ') for subval in reference])
```

Comment:
@czue I added this in operators dict https://github.com/dimagi/commcare-hq/pull/8842/files#diff-0da33b434582bbcf21b2db76d7da5b6aR20